src/component/TodoForm.jsx

```diff
@@ -0,0 +1,50 @@
+    import React, { useState } from 'react'
+    import Button from './Button'
+    
+    const ToDoForm = ({onAddTodo}) => {
+
+
+        const [TaskName, setTodoText] = useState('');
+        const [TaskStatus, setTodoStatus]=useState('todo');
+
+        const maxlength = 25;
+
+         const handleSubmit = (e) => {
+            e.preventDefault();
+            if(TaskName.trim()) {
+                onAddTodo({text:TaskName, status:TaskStatus});
+                setTodoText('')
+                setTodoStatus('Todo')
+            }
+         }
+        
+        const handleInputChange = (e) =>{
+           setTodoText(e.target.value)
+        }
+
+        const handleStatusChange = (e) => {
+          setTodoStatus (e.target.value)
+        }
+
+        const isButtonDisabled = TaskName.trim() === '';
```

Comment:
There is no need of a new variable for this we can directly use it like this 'isDisabled={!taskName.trim()}'